src/zui/ZUIEventParticipantLists.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC } from 'react';
+import ZUINumberChip from './ZUINumberChip';
+
+import { Box, Typography } from '@mui/material';
+import { DataGridPro, GridColDef } from '@mui/x-data-grid-pro';
+import {
+  ZetkinEventParticipant,
+  ZetkinEventResponse,
+} from 'utils/types/zetkin';
+
+interface ZUIEventParticipantsListProps {
+  chipColor: string;
+  chipNumber: string;
+  columns: GridColDef[];
+  description: string;
+  rows: ZetkinEventResponse[] | ZetkinEventParticipant[];
+  title: string;
+}
+
+const ZUIEventParticipantsList: FC<ZUIEventParticipantsListProps> = ({
+  chipColor,
+  chipNumber,
+  columns,
+  description,
+  rows,
+  title,
+}) => {
+  return (
+    <>
+      <Box
+        sx={{
+          '& div': { backgroundColor: 'transparent' },
+          alignItems: 'center',
+          display: 'flex',
+          flexDirection: 'row',
+          marginBottom: '15px',
+          marginTop: '15px',
+        }}
+      >
+        <Typography mr={2} variant="h4">
+          {title}
+        </Typography>
+        <ZUINumberChip color={chipColor} outlined={true} value={chipNumber} />
+      </Box>
+      <Typography mb={2} variant="body1">
+        {description}
+      </Typography>
+      <DataGridPro
+        autoHeight
+        checkboxSelection
+        columns={columns}
+        rows={rows ?? []}
+      />
+    </>
+  );
+};
+
+export default ZUIEventParticipantsList;
```

Comment:
The name of this module and the name of the component do not match. The component is called `ZUIEventParticipantsList` (plural participants, singular list) but the module is called `ZUIEventParticipantLists` (singular participant, plural lists).

I also don't think this should be a ZUI component since I doubt that it would be used anywhere outside of the events feature. I would even just call it `ParticipantListSection` and put it in in the same module as where it's used, i.e. in `EventParticipantsList`.